CHANGELOG.md

```diff
@@ -4,6 +4,224 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.6.0] - 2022-12-13
+
+## ❗ BREAKING ❗
+
+### Protoc now required to build ([Issue #1970](https://github.com/apollographql/router/issues/1970))
+
+Protoc is now required to build Apollo Router. Upgrading to Open Telemetry 0.18 has enabled us to upgrade tonic which in turn no longer bundles protoc.
+Users must install it themselves https://grpc.io/docs/protoc-installation/.
+
+By [@bryncooke](https://github.com/bryncooke) in https://github.com/apollographql/router/pull/1970
+
+### Jaeger scheduled_delay moved to batch_processor->scheduled_delay ([Issue #2232](https://github.com/apollographql/router/issues/2232))
+
+Jager config previously allowed configuration of scheduled_delay for batch span processor. To bring it in line with all other exporters this is now set using a batch_processor section.
+
+Before:
+```yaml
+telemetry:
+  tracing:
+    jaeger:
+      scheduled_delay: 100ms
+```
+
+After:
+```yaml
+telemetry:
+  tracing:
+    jaeger:
+      batch_processor:
+        scheduled_delay: 100ms
+```
+
+By [@bryncooke](https://github.com/bryncooke) in https://github.com/apollographql/router/pull/1970
+
+## 🚀 Features
+
+### Add support for experimental tooling ([Issue #2136](https://github.com/apollographql/router/issues/2136))
+
+Display a message at startup listing used `experimental_` configurations with related GitHub discussions.
+It also adds a new cli command `router config experimental` to display all available experimental configurations.
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/2242
+
+### Re-deploy router pods if the SuperGraph configmap changes ([PR #2223](https://github.com/apollographql/router/pull/2223))
+When setting the supergraph with th the `supergraphFile` variable a `sha256` checksum is calculated and set as an annotation for the router pods. This will spin up new pods when the supergraph is mounted via config map and the schema has changed.
```

Comment:
```suggestion
When setting the supergraph with the `supergraphFile` variable a `sha256` checksum is calculated and set as an annotation for the router pods. This will spin up new pods when the supergraph is mounted via config map and the schema has changed.
```